src/mmw/js/src/modeling/templates/scenarioMenuItemOptions.html

```diff
@@ -0,0 +1,30 @@
+{% if not is_new %}
+    <li class="scenario-options-dropdown-menu-item" role="presentation" role="presentation" data-action="share">
+        <i  class="fa fa-share-alt" title="Share"></i> Share
+    </li>
+{% endif %}
+{% if not is_current_conditions %}
+    <li class="scenario-options-dropdown-menu-item" role="presentation" li data-action="duplicate">
+        <i class="fa fa-copy" title="Duplicate"></i> Duplicate
+    </li>
+{% endif %}
+{% if not is_current_conditions %}
+    <li class="scenario-options-dropdown-menu-item" role="presentation" data-action="delete">
+        <i class="fa fa-trash" title="Delete"></i> Delete
+    </li>
+{% endif %}
+{% if is_gwlfe %}
+    <li class="scenario-options-dropdown-menu-item" role="presentation" data-action="export-gms">
+        <form id="export-gms-form" method="post" action="/api/modeling/export/gms/" target="_blank">
+            <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrftoken }}">
+            <input type="hidden" name="filename" class="gms-filename" value="export.gms">
+            <input type="hidden" name="mapshed_data" value='{{ gis_data }}'>
```

Comment:
Ah 🤦‍♂️ serves me right for being too nitpicky. And of course I'm the one who wrote this originally https://github.com/WikiWatershed/model-my-watershed/commit/2834f5ba0093ca968d1fe9fe744d2e0a9687c567